doc/notebooks/jupyter_integration.ipynb

```diff
@@ -86,17 +86,38 @@
     "# Demonstration of GrassRenderer for non-interactive map display\n",
     "m = gj.GrassRenderer(height=540, filename = \"streams_map.png\")\n",
     "\n",
-    "m.run(\"d.rast\", map=\"elevation\")\n",
-    "m.run(\"d.vect\", map=\"streams\")\n",
+    "# Set computational region to the study area.\n",
+    "gs.run_command(\"g.region\", raster=\"elevation\", flags=\"p\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Non-interactive Map Display"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Demonstration of GrassRenderer for non-interactive map display\n",
     "\n",
-    "m.show()"
+    "r = gj.GrassRenderer(height=540, filename = \"streams_maps.png\")\n",
```

Comment:
There is a `m = gj.GrassRenderer` line above the g.region line. So delete that one?